apps/web/lib/validators/escrow.ts

```
@@ -229,3 +233,175 @@ export function validateMilestoneReview(
+		!['APPROVED', 'REJECTED', 'RESOLVED'].includes(data.resolution)
+	) {
+		errors.push('Invalid resolution status.')
+	}
+
+	// Validate resolutionNotes
+	if (!data.resolutionNotes?.trim()) {
+		errors.push('Resolution notes are required.')
+	}
+
+	// Validate approverAmount
+	if (!data.approverAmount || isNaN(Number(data.approverAmount))) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)